incubator/tensorflow-inception/templates/inception-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "fullname" . }}-seed-inception
```

Comment:
They're in the best practices but optional: https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md. It's always good to have at least one label that can be used to query resources from a chart